recipes/stellar/meta.yaml

```diff
@@ -17,5 +17,6 @@ test:
     - stellar --help 2>&1 > /dev/null
 about:
   home: https://github.com/seqan/seqan/tree/master/apps/stellar/README
-  license: https://github.com/seqan/seqan/tree/master/apps/stellar/LICENSE
+  license: LGPLv3+
+  license_file: https://github.com/seqan/seqan/tree/master/apps/stellar/LICENSE
```

Comment:
The filename is relative to the source directory.
https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#license-file

Reply:
just pushed source reference